File: 3242-count-elements-with-maximum-frequency/count-elements-with-maximum-frequency.py
class Solution:
    def maxFrequencyElements(self, nums: List[int]) -> int:
        mp = {}
        for x in nums:
            mp[x] = mp.get(x, 0) + 1
        maxi = 0
        for key, value in mp.items():
            maxi = max(maxi, value)
        ans = 0
        for key, value in mp.items():
            if value == maxi:
                ans += value
        return ans

File: 3242-count-elements-with-maximum-frequency/count-elements-with-maximum-frequency.cpp
class Solution {
public:
    int maxFrequencyElements(vector<int>& nums) {
        unordered_map<int,int>freq;
        for(auto i : nums){
            freq[i]++;
        }
        int maxi = 0;
        for(auto i : freq){
            if(i.second > maxi){
                maxi = i.second;
            }
        }
        int ans = 0;
        for(auto i : freq){
            if(i.second == maxi){
                ans += maxi;
            }
        }
        return ans;
    }
};